### ISSUE TITLE
[FR] custom log note

### ISSUE THREAD

**Issue by jeetsukumaran** [franciskafieh/obsidian-list-modified#79]
## First Steps

-   [ x] I've ensured that this feature request **does not already exist** by [using the search issues feature](https://github.com/franciskafieh/obsidian-list-modified/issues?q=search+issues+here)
-   [ x] I've read the documentation (see [the README](https://github.com/franciskafieh/obsidian-list-modified#readme))
-   [ ]x I'm on the latest release of obsidian-list-modified

## Describe the desired feature

Is it possible to decouple the destination from the daily note? It would be nice to have the change file logs in a different file from, say, capture and tasks etc. Basically, a custom folder/name pattern destination for just the changed file logs, e.g.

```
[archives/change-logs/]YYYY/YYYY-MM-DD[.changed-notes.log]
```

**Comment by franciskafieh** [comment]
Actually, I'm going to reopen #33 and merge this into there. Please see my comment there for a current workaround in regards to graph (if that's what you're after)

**Comment by jeetsukumaran** [comment]
Hi, just a quick follow up. #33 does not apply here -- my wish is to separate the content, not the links. In particular, I don't want the list of changed files to be injected in my designated daily note. I would like the entire "changed files" log to be stored in a different set of files, also timestamped, in parallel with my journal/capture/daily note set of files.

**Comment by franciskafieh** [comment]
>  I would like the entire "changed files" log to be stored in a different set of files, also timestamped, in parallel with my journal/capture/daily note set of files.

from what I understand, a suffix would still suffice for this. E.g. having 2023-04-09 as your daily note, and 2023-04-09-links as your "suffix" note

**Comment by jeetsukumaran** [comment]
Oh, you are right. That would solve the problem very well. Thanks :)
